src/Apps/IdentityVerification/IdentityVerificationApp/index.tsx

```diff
@@ -108,7 +106,7 @@ const IdentityVerificationApp: React.FC<Props> = ({ me, relay, router }) => {
   }
 
   const handleMutationSuccess = (identityVerificationFlowUrl: string) => {
-    router.push(identityVerificationFlowUrl)
+    window.location.assign(identityVerificationFlowUrl)
```

Comment:
What is `identityVerificationFlowUrl`?

Reply:
it's an external URL to jumio- https://artsy.netverify.com/adsfasdf